taskmanager/src/Infrastructure/InMemory/ReportRegistry.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types=1);
```

Comment:
I'm not sure that we need store reports - generate `in fly` should be enough

Reply:
@tarnawski removed.